tests/test_download_manager.py

```
@@ -0,0 +1,478 @@
+        task = dm.start_download("dl-1", "http://example.com/model.bin", dest)
+        assert task.id == "dl-1"
+        assert task.url == "http://example.com/model.bin"
+        assert task.dest == dest
+        assert task.status == "pending"
+        assert "dl-1" in dm._tasks
+        assert "dl-1" in dm._running
+        dm._running["dl-1"].cancel()
+        try:
+            await dm._running["dl-1"]
+        except (asyncio.CancelledError, Exception):
+            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)